Multisig/UI/Transactions/TransactionViewModel/TransactionViewModel.swift

```diff
@@ -41,7 +44,10 @@ class TransactionViewModel: Identifiable {
 
     init(_ tx: Transaction, _ safe: SafeStatusRequest.Response) {
         id = UUID()
+        hash = tx.transactionHash?.description
         date = tx.executionDate ?? tx.submissionDate ?? tx.modified
+        createdDate = tx.submissionDate.map { Self.dateFormatter.string(from: $0) } ?? ""
+        executedDate = tx.executionDate.map { Self.dateFormatter.string(from: $0) } ?? ""
```

Comment:
this line is good fro the base view model, but it does not count for the "TransferTransactionViewModel" because the "TransactionTransfer" struct has its own "executionDate", which, I think, should be used for that view model. (see TransferTransactionViewModel.swift line 86)

Reply:
resolved, the yare the same ✅